### ISSUE TITLE
Bug: russian commands doesn't work properly

### ISSUE THREAD

**Issue by op-hui** [evennia/evennia#799]
#### Steps to reproduce the issue:

1.  Create command with russian utf-8 characters
2.  Try use command in game. For example command name is "команда", applying for player "player"
3.   U - user input, S - server side
```
U: команда player 
S: Could not find 'ayer'.
U: команда  player #  TWO spaces between command and playername
S: Could not find 'layer'. 
U: команда   player # THREE spaces between command and playername
S: ok
```
If put THREE spaces between utf-8 russian command name and plaername, they will fork fine

#### What I expect to see and what I actually see (tracebacks, error messages etc):
I see: команда player -> Could not find 'ayer'
I expected: команда player -> ok
No error messages in log

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve:

```
$ evennia --version

    Evennia 0.5.0 (rev a1eadef) (rev a1eadef)
    OS: posix
    Python: 2.7.6
    Twisted: 15.4.0
    Django: 1.8.4

```

**Comment by Griatch** [comment]
@op-hui: Could you also post the source code for your "команда" command?

**Comment by op-hui** [comment]
```python
class CmdTest(Command):
        
        key = "команда"
        aliases = ["команда2"] 
        locks = "cmd:all()"
        help_category = "General"

        def parse(self):
            "Very trivial parser" 
            self.target = self.args.strip() 

        def func(self):
            "This actually does things"
            caller = self.caller
            if not self.target or self.target == "here":
                string = "%s команда." % caller.name
                caller.location.msg_contents(string, exclude=caller)
                caller.msg("Ты команда")
            else:
                target = caller.search(self.target)
                if not target: 
                    # caller.search handles error messages
                    return
                string = "%s команда на тебя." % caller.name
                target.msg(string)
                string = "Ты команда на %s." % target.name
                caller.msg(string)
                string = "%s команда %s." % (caller.name, target.name)           
                caller.location.msg_contents(string, exclude=[caller,target])
```

**Comment by Griatch** [comment]
@op-hui I can't test this right now, but what do you get if you do `print self.target, self.args` (prints will appear in your log/terminal) before going into `caller.search`?

**Comment by op-hui** [comment]
Got it, put debug before caller.search
i'm trying apply команда to anonymous player
and got in logs "2015-09-04 10:29:46+0000 [-] onymous onymous"

**Comment by op-hui** [comment]
Bug presented in webclient and  telnet sessions

**Comment by op-hui** [comment]
Between command named 'команда' and player name (named anonymous) i can put ANY THREE CHARACTERS:
```
>командаXXXplayer
Ok
>командаYYYplayer
Ok
>командаYYplayer
Could not find 'layer'.
```

**Comment by Griatch** [comment]
Thanks for the further info. My first guess is that his may be a regex error in Evennia's command parser, possibly it is not setting the `UNICODE` flag correctly somewhere. Will look at it tonight.

**Comment by Griatch** [comment]
This is resolved in a7e42b2a5bece08fa5a1ae179b3835a19cc53c7e. Closing.
